charts/rancher-monitoring/rancher-monitoring-crd/14.5.101+up14.5.0/templates/jobs.yaml

```diff
@@ -45,7 +45,18 @@ spec:
           - /bin/sh
           - -c
           - >
-            kubectl apply -f /etc/config/crd-manifest.yaml
+            kubectl apply -f /etc/config/crd-manifest.yaml;
+            echo "Waiting for CRDs to be recognized before finishing installation...";
+            
+            {{- range $path, $_ :=  (.Files.Glob  "crd-manifest/**.yaml") }}
+            {{- $apiGroup := get (get ($.Files.Get $path | fromYaml) "spec") "group" }}
+            rm -rf $HOME/.kube/cache/discovery/*/{{ $apiGroup }}
+            {{- end }}
+
+            {{- range $path, $_ :=  (.Files.Glob  "crd-manifest/**.yaml") }}
+            {{- $crd := get (get ($.Files.Get $path | fromYaml) "metadata") "name" }}
+            kubectl get crd {{ $crd }}
```

Comment:
This call can/will fail, if we are attempting to duplicate what helm does it should be an error check and retry loop until the CRDs are available through the API 
https://github.com/helm/helm/blob/43853ea772e779b617b011a8d4e69205636fa4f9/pkg/kube/wait.go#L52:18